res/xml/latn_bepo_fr.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Inspired by the bépo layout, but adapted to smartphone -->
+<keyboard name="BEPO (Français)" script="latin">
+<row>
+    <key width="1.1" key0="b" key1="esc" key2="1" key4="|"/>
+    <key width="1.1" key0="é" key2="2" key4="è"/>
+    <key width="1.1" key0="p" key2="3"/>
+    <key width="1.1" key0="o" key2="4" key3="œ" key4="accent_circonflexe"/>
+    <key width="1.1" key0="v" key2="5" key4="{"/>
+    <key width="1.1" key0="d" key2="6" key4="}"/>
+    <key width="1.1" key0="l" key2="7" key4="("/>
+    <key width="1.1" key0="j" key2="8" key4=")"/>
+    <key width="1.1" key0="z" key2="9" key4="["/>
+    <key width="1.1" key0="w" key2="0" key4="]"/>
+  </row>
+  <row>
+    <key width="1.1" key0="a" key1="tab" key3="æ" key4="à"/>
+    <key width="1.1" key0="u" key1="accent_trema" key4="$"/>
+    <key width="1.1" key0="i" key4="&amp;"/>
+    <key width="1.1" key0="e" key1="€" key2="accent_aigu" key4="accent_grave"/>
```

Comment:
Are `accent_aigu` and `accent_grave` needed ?

Reply:
only for ù I think, so I can add ù directly in the keyboard and remove ```accent_aigu``` and ```accent_aigu```

Reply:
Note that if you remove the accent dead keys, they will still appear on the keyboard. This is a known issue that I intent to solve before the next release.